core-bundle/src/Search/Backend/Provider/TableDataContainerProvider.php

```diff
@@ -202,7 +202,13 @@ private function findDocuments(string $table, ReindexConfig $reindexConfig): \Ge
 
         $searchableFields = array_filter(
             $fieldsConfig,
-            static fn (array $config): bool => isset($config['search']) && true === $config['search'],
+            static function (array $config): bool {
+                if (\array_key_exists('backendSearch', $config)) {
+                    return $config['backendSearch'];
```

Comment:
```suggestion
                    return true === $config['backendSearch'];
```

Matching the same as in L210 for readability 🤔?

Reply:
We could, yes. Leaving that up to @leofeyer as CS :)

Reply:
It's not CS, though, but the decision if truthy/falsy values are allowed or only boolean. For me it would be OK to have the simple version and still expect all definitions to be boolean. But we should keep it consistent, otherwise you would ask yourself this question everytime when reading the code. :smile:

Reply:
We have tons of configuration fields that expect to be `true` or `false` and we only ever check using `isset()` so...
I'm not going to update this PR.

Reply:
Did it anyway.